### ISSUE TITLE
Question Pool

### ISSUE THREAD

**Issue by DanielHamar** [DanielHamar/DAB_Advanced_JAVA#3]
Create a pool of questions based on Math, History, Science, Geography, and English.  Pool should consist of 20 to 25 questions per subject.  Questions should have full Multiple Choice aspects with 3 to 4 possible answers and only one correct answer.

**Comment by chiefhamar** [comment]
This will not be completely done by the June 26 milestone.  I've started and I have the sites I will pull the questions from, but I will need more time to actually get them typed out in a usable format.  Will keep you posted.

**Comment by chiefhamar** [comment]
I've updated the share folder with the newest changes to the multiple choice portion.  The updates include all the questions.  I've finished putting the questions and answers together for five subjects.  I got rid of geography and replaced it with Social Studies.  When running the program, simply change the file name (math.txt - science.txt - social_studies.txt - history.txt - english.txt) to see the different questions.  I won't promise that it's bug free, but its pretty close.  I can make changes based on feedback if you guys have any.  I will be starting the main menu next.
